gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionExtension.java

```diff
@@ -59,7 +59,15 @@ public JavaServiceDistributionExtension(Project project) {
         javaVersion = objectFactory.property(JavaVersion.class).value(project.provider(() ->
                 project.getConvention().getPlugin(JavaPluginConvention.class).getSourceCompatibility()));
         mainClass = objectFactory.property(String.class);
-        javaHome = objectFactory.property(String.class);
+
+        javaHome = objectFactory.property(String.class).value(javaVersion.map(javaVersionValue -> {
+            boolean javaVersionLessThan11 = javaVersionValue.compareTo(JavaVersion.VERSION_11) < 0;
+            if (javaVersionLessThan11) {
+                return "";
```

Comment:
Returning empty string here because that was what was happening in:
https://github.com/palantir/sls-packaging/blob/5266eb5bca9beb40c13988ff2ec74090f6b79aed/gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java#L162

Ideally I'd make this provider somehow indicate it's empty, but I can't see how to do that and keep this lazy in Gradle.

Reply:
our internal deployment daemon has the implicit assumption of Java 8 anywhere - rather than a < Java11 check, shall we do <= Java8?